README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
```

Comment:
> I do want to do a pgbench example, but I had an idea today to do that with a Docker image so I want to try that first.

All of that is already done, it just needs to be made into real documentation rather than notes: https://github.com/disqus/pgshovel/blob/replication/example/README.txt